pkg/collector/scheduler.go

```diff
@@ -166,6 +165,7 @@ func (s *CheckScheduler) getChecks(config integration.Config) ([]check.Check, er
 		if instanceConfig.LoaderName != "" {
 			selectedInstanceLoader = instanceConfig.LoaderName
 		}
+		log.Debugf("Check '%s' using loader %s (init_config loader: %s, instance loader: %s)", config.Name, selectedInstanceLoader, initConfig.LoaderName, instanceConfig.LoaderName)
```

Comment:
This log line could be a little confusing if no specific loader has been configured (it would just log lots of empty values for the loader names correct?). I'd replace this with something like:

```suggestion
		if (selectedInstanceLoader != "") {
			log.Debugf("Loading check instance for check '%s' using loader %s (init_config loader: %s, instance loader: %s)", config.Name, selectedInstanceLoader, initConfig.LoaderName, instanceConfig.LoaderName)
		} else {
			log.Debugf("Loading check instance for check '%s' using default loaders", config.Name)
		}
```

wdyt?

Reply:
Yes, that's a good point, thx for the suggestion. Accepting your suggestion and testing that locally.